brainglobe_utils/image_io/load.py

```diff
@@ -197,6 +198,9 @@ def load_img_stack(
     logging.debug(f"Loading: {stack_path}")
     stack = tifffile.imread(stack_path)
 
+    if stack.ndim != 3:
+        raise ImageIOLoadException(error_type="2D tiff")
```

Comment:
Could we add the supported data types to the docstring, and specify that 2D tiffs, and paths to folders containing a single tiff or differently shaped tiffs are not supported?

[Edit] Or (better?) specify this by linking to the `ImageIOLoadException` docs?

Reply:
@alessandrofelder I've linked to the `ImageIOLoadException` docs for the `load_any` function. For the rest, as only some of the errors in `ImageIOLoadException` are relevant, I've updated the docstrings directly to make this clearer. Could you take a quick look, and merge if you're happy with the new docstrings?